mapswipe_workers/mapswipe_workers/mapswipe_workers.py

```diff
@@ -406,6 +406,36 @@ def run_delete_project(project_id, project_ids):
         click.echo("Invalid input")
 
 
+@cli.command("set-tileserver-api-key")
+@click.option(
+    "--project-id",
+    "-i",
+    help="Set api key for project with given project id",
+    type=str,
+)
+@click.option(
+    "--project-ids",
+    cls=PythonLiteralOption,
+    default="[]",
+    help=(
+        "Set api key for multiple projects. "
+        "Provide project id strings as a list: "
+        """'["project_a", "project_b"]'"""
+    ),
+)
+@click.option("--api-key", help="the new api key to use", type=str, required=True)
+def run_set_tileserver_api_key(project_id, project_ids, api_key):
+    """Change the imagery API key for a project in Firebase."""
+    if not project_ids and not project_id:
+        click.echo("Missing argument")
+        return None
+    elif not project_ids:
+        project_ids = [project_id]
+
+    for project_id in project_ids:
+        update_data.set_tileserver_api_key(project_id, api_key)
```

Comment:
Is there some easy check you could run on the `api_key` argument to help catch typos and stuff? Maybe check the length if it's known, or ideally, run a test request with the new api key to check that the server accepts it.